servers/rendering_server.cpp

```diff
@@ -438,13 +438,14 @@ Error RenderingServer::_surface_set_data(Array p_arrays, uint32_t p_format, uint
 				ERR_FAIL_COND_V(array.size() != p_vertex_array_len, ERR_INVALID_PARAMETER);
 
 				const Color *src = array.ptr();
-				uint16_t color16[4];
 				for (int i = 0; i < p_vertex_array_len; i++) {
-					color16[0] = Math::make_half_float(src[i].r);
-					color16[1] = Math::make_half_float(src[i].g);
-					color16[2] = Math::make_half_float(src[i].b);
-					color16[3] = Math::make_half_float(src[i].a);
-					memcpy(&aw[p_offsets[ai] + i * p_attrib_stride], color16, 8);
+					uint8_t color8[8] = {
```

Comment:
Shouldn't this be `color8[4]`? What are the 4 remaining fields used for?

Reply:
yeah my bad